components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/admin/v1/dto/KeyManagerDTO.java

```diff
@@ -339,7 +339,7 @@ public KeyManagerDTO alias(String alias) {
   }
 
   
-  @ApiModelProperty(example = "ZyxWExFdfe4fAZ7sc1W8bkX2r80a", value = "The alias of Identity Provider. If the tokenType is EXCHANGED, the alias value should be equal to the audience value of the JWT token ")
+  @ApiModelProperty(example = "https://localhost:8243/token", value = "The alias of Identity Provider. If the tokenType is EXCHANGED, the alias value should be equal to the audience value of the JWT token ")
```

Comment:
value should be inclusive in audience ( it can be one of audience )

Reply:
Fixed via 446173df7a595cd00744cb964c8a6f7597b5d274